src/shared/models/game/GameSetup.es6

```diff
@@ -0,0 +1,83 @@
+// const BaseError = require('../error/BaseError')
+const Player = require('./Player')
+const Market = require('./Market')
+const Tile = require ('./Tile')
+// const GameState = require ('./GameState')
+const _ = require('underscore')
+
+
+const GameSetup = {
```

Comment:
So will this represent just the starting state? or is the gamesetup used to represent the board throughout the game?